libcxx/test/libcxx/type_traits/is_replaceable.compile.pass.cpp

```diff
@@ -0,0 +1,269 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include <__type_traits/is_replaceable.h>
+#include <array>
+#include <deque>
```

Comment:
Do you know why the container adaptors `stack`, `queue` and `priority_queue` are not considered for `trivial_foo`?

Reply:
I don't, but @philnik777 might. I added support for `is_replaceable` for the same set of types we already handled for `is_trivially_relocatable`, no more. I suggest we expand it in future patches.